src/app/screens/LNURLPay.tsx

```diff
@@ -130,7 +130,7 @@ function LNURLPay(props: Props) {
 
   function renderAmount() {
     if (details.minSendable === details.maxSendable) {
-      return <p>{`${details.minSendable / 1000} Satoshi`}</p>;
+      return <p>{`${details.minSendable / 1000} sat`}</p>;
```

Comment:
not sure where this function is used in the extension. I would keep Satoshi if there is no amount mentioned before.

Reply:
in this case there is an amount `{`${details.minSendable / 1000}` that is basically code that inserts an amount.